engine/runtime/src/main/java/org/enso/interpreter/runtime/error/DataflowError.java

```diff
@@ -49,9 +42,11 @@ public final class DataflowError extends AbstractTruffleException implements Ens
    * @param location the node in which the error was created
    * @return a new dataflow error
    */
-  public static DataflowError withoutTrace(Object payload, Node location) {
+  public static DataflowError withDefaultTrace(State state, Object payload, Node location) {
     assert payload != null;
-    if (assertsOn) {
+    boolean attachFullStackTrace =
```

Comment:
There is a regression in standard library benchmarks
- #10702 

My hypothesis: It has to be caused the call to `hasContextEnabled` check on this line.